root/edit/details/AddPlace.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * @flow
+ * Copyright (C) 2019 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import {formatCoordinates} from '../../utility/coordinates';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AddPlaceEditT = {
+  ...EditT,
+  display_data: {
+    ...DatePeriodRoleT,
+    address: string | null,
+    area: AreaT,
+    comment: string | null,
+    coordinates: CoordinatesT,
+    name: string,
+    place: PlaceT,
+    type: PlaceTypeT,
+  },
+};
+
+const AddPlace = ({edit}: {edit: AddPlaceEditT}) => {
+  const display = edit.display_data;
+  const place = display.place;
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Place'))}</th>
+            <td><DescriptiveLink allowNew entity={place} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-place">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Name'))}</th>
+            <td>{lp_attributes(display.name, 'place_type')}</td>
```

Comment:
This is the actual name of the place, it doesn't need translating.

Reply:
Corrected it thanks :)